goosebit/api/v1/software/routes.py

```diff
@@ -52,7 +55,7 @@ async def software_delete(_: Request, delete_req: SoftwareDeleteRequest) -> Stat
     "",
     dependencies=[Security(validate_user_permissions, scopes=["software.write"])],
 )
-async def post_update(_: Request, file: UploadFile = File(None), url: str = Form(None)):
+async def post_update(_: Request, file: Optional[UploadFile] = File(None), url: Optional[str] = Form(None)):
```

Comment:
I think we prefer `T | None` over `Optional[T]` generally.

Reply:
Cleaned up this code and the other places where Optional was used.